### PR TITLE
Simulations tests

### PR DESCRIPTION
**Pull request by Kevin2**
adding tests for simulations
standard image size is changed to 63 x 63 for symmetry reasons

fixing issue #42

### CODE REVIEW

File: simulations/uv_simulations.py [hunk 1]
```diff
@@ -176,6 +175,11 @@ def get_baselines(self):
             y = baselines[1::2]
             x_base = np.append(x_base, x)
             y_base = np.append(y_base, y)
+            
+        drops = np.asarray([((i * 2 + np.array([1, 2])) - 1) + (i * self.len*2) for i in range(self.len)])
+        coords = np.delete(np.stack([x_base, y_base], axis=1), drops.ravel(), axis=0).T
+        x_base = coords[0]
+        y_base = coords[1]
```
Comment (FritzGeise) [inline simulations/uv_simulations.py]: Warum ist das nötig?

File: simulations/uv_simulations.py [hunk 2]
```diff
@@ -188,6 +192,9 @@ def to_enu(self, x_ref, y_ref, z_ref):
             x, y, z reference coordinates
         """
         lon_ref, lat_ref = self.to_geodetic(x_ref, y_ref, z_ref, enu=True)
+        if isinstance(x_ref, int):
+            x_ref, y_ref, z_ref = [x_ref], [y_ref], [z_ref]
+            lon_ref, lat_ref = [lon_ref], [lat_ref]
```
Comment (FritzGeise) [inline simulations/uv_simulations.py]: Warum ist das nötig?

File: tests/test_uv_plots.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import os
+
+
+def test_plot_uv_coverage():
+    from simulations.uv_plots import plot_uv_coverage
+
+    u = np.array(range(5))
+    v = np.array(range(5))
+
+    assert plot_uv_coverage(u, v) is None
```
Comment (FritzGeise) [inline tests/test_uv_plots.py]: Was genau fragt man mit diesem `is None` ab?

File: simulations/gaussian_simulations.py [hunk 4]
```diff
@@ -163,6 +163,7 @@ def create_gaussian_source(
     components should not have too big gaps between each other
     """
     if sides == 1:
+        print(amp)
```
[the remark this answers is not in the document]
Comment (Kevin2) [reply simulations/gaussian_simulations.py]: good point, I will delete this

File: simulations/uv_simulations.py [hunk 5]
```diff
@@ -87,8 +87,8 @@ def propagate(self, num_steps=None, multi_pointing=False):
             raise ValueError("Length of lon and lat are different!")
 
         if multi_pointing is True:
-            lon = self.mod_delete(lon, 5, 20)
-            lat = self.mod_delete(lat, 5, 20)
+            lon = self.mod_delete(lon, 5, 10)
+            lat = self.mod_delete(lat, 5, 10)
```
[the remark this answers is not in the document]
Comment (Kevin2) [reply simulations/uv_simulations.py]: We did not use this option yet. I tested it and realised that the step of 20 points was a bit to big.

File: simulations/uv_simulations.py (hunk 1 above)
Comment (Kevin2) [reply simulations/uv_simulations.py]: I delete all 0 length baselines. Before there were baselines from 1 antenna to the same antenna, which did not make sense.

File: simulations/uv_simulations.py (hunk 2 above)
Comment (Kevin2) [reply simulations/uv_simulations.py]: This allows to do the calculation also for integer inputs. Before the calculations were only possible when your inputs where arrays of x_ref, y_ref and z_ref values.

File: tests/test_uv_plots.py (hunk 3 above)
Comment (Kevin2) [reply tests/test_uv_plots.py]: The plot function is called. As it does not return anything, we expect a none as result. If there is a problem during plotting an error will be raised and the test fails.

**Pull request merged**